src/udiskslinuxblock.c

```diff
@@ -2727,6 +2746,20 @@ udisks_linux_block_is_unknown_crypto (UDisksBlock *block)
 
 /* ---------------------------------------------------------------------------------------------------- */
 
+void
+udisks_linux_block_set_encrypted_lock (UDisksBlock *block,
+                                       gboolean    lock)
```

Comment:
I think it would be a lot nicer to have `lock/unlock` or `acquire/release` methods.

Reply:
Ok, fixed.